src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -139,7 +139,9 @@
 using namespace std;
 using namespace ngraph;
 
-static bool s_use_ref_kernels = (std::getenv("NGRAPH_CPU_USE_REF_KERNELS") != nullptr);
+static const bool s_use_ref_kernels = (std::getenv("NGRAPH_CPU_USE_REF_KERNELS") != nullptr);
+
+static const size_t s_one = 1;
```

Comment:
Can we get a comment that this is needed for MSVC compiler or whatever the reason we need this?

Reply:
Thank you for the idea, done.